apps/web/utils/typeChecking.ts

```diff
@@ -12,3 +12,7 @@ export const isArray = (value: unknown): value is unknown[] => {
 export const isObject = (value: unknown): value is Record<string, unknown> => {
   return typeof value === "object" && !ThisModule.isArray(value);
 };
+
+export const isNumber = (value: unknown): value is number => {
```

Comment:
Thinking this could be `isPositiveNumber` and not allow negatives since I don't think any of our use cases should allow negatives.

Reply:
I added a minValue to the component itself so we don't have to deal with negative numbers at all. Users can't input them because they don't make sense.